ajax/libs/WhitestormJS/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "WhitestormJS",
+  "filename": "whitestorm.js",
+  "version": "1.2.6",
+  "homepage": "https://whsjs.readme.io/docs",
+  "author": "sasha240100 <siteprogcom@gmail.com>",
```

Comment:
@sasha240100 Thanks for help! And yes I think so. The auto-update mechanism will let users fetch the latest version if it has a valid tag.